package.json

```diff
@@ -46,7 +46,8 @@
     "ember-resolver": "^4.5.5",
     "ember-source": "2.18.2",
     "eslint-plugin-ember-suave": "^1.0.0",
-    "loader.js": "^4.6.0"
+    "loader.js": "^4.6.0",
+    "sass": "^1.77.6"
```

Comment:
Add details and screenshots explaining the need for this change

Reply:
https://www.npmjs.com/package/ember-cli-sass/v/8.0.1